### PR TITLE
Update what's new for .NET 9 Preview 5

### LINKED ISSUE #39133: Update What's new in .NET 9 for Preview 5
---
[Associated WorkItem - 259837](https://dev.azure.com/msft-skilling/Content/_workitems/edit/259837)

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #39133.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/whats-new/dotnet-8/runtime.md](https://github.com/dotnet/docs/blob/0f06c835a097a59b7ffae63148816d2034cd0883/docs/core/whats-new/dotnet-8/runtime.md) | [What's new in the .NET 8 runtime](https://review.learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-8/runtime?branch=pr-en-us-41357) |
| [docs/core/whats-new/dotnet-9/libraries.md](https://github.com/dotnet/docs/blob/0f06c835a097a59b7ffae63148816d2034cd0883/docs/core/whats-new/dotnet-9/libraries.md) | [What's new in .NET libraries for .NET 9](https://review.learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-9/libraries?branch=pr-en-us-41357) |
| [docs/core/whats-new/dotnet-9/overview.md](https://github.com/dotnet/docs/blob/0f06c835a097a59b7ffae63148816d2034cd0883/docs/core/whats-new/dotnet-9/overview.md) | [What's new in .NET 9](https://review.learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-9/overview?branch=pr-en-us-41357) |
| [docs/core/whats-new/dotnet-9/runtime.md](https://github.com/dotnet/docs/blob/0f06c835a097a59b7ffae63148816d2034cd0883/docs/core/whats-new/dotnet-9/runtime.md) | [docs/core/whats-new/dotnet-9/runtime](https://review.learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-9/runtime?branch=pr-en-us-41357) |
| [docs/core/whats-new/dotnet-9/sdk.md](https://github.com/dotnet/docs/blob/0f06c835a097a59b7ffae63148816d2034cd0883/docs/core/whats-new/dotnet-9/sdk.md) | [What's new in the SDK for .NET 9](https://review.learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-9/sdk?branch=pr-en-us-41357) |

### CODE REVIEW

File: docs/core/whats-new/dotnet-8/runtime.md [hunk 1]
```diff
@@ -777,9 +777,7 @@ The new <xref:Microsoft.Extensions.Diagnostics.Metrics.Testing.MetricCollector%6
...
 
-AI workloads like semantic search and retrieval-augmented generation (RAG) extend the natural language capabilities of large language models such as ChatGPT by augmenting prompts with relevant data. For these workloads, operations on vectors&mdash;like *cosine similarity* to find the most relevant data to answer a question&mdash;are crucial. The System.Numerics.Tensors.TensorPrimitives package provides APIs for vector operations, meaning you don't need to take an external dependency or write your own implementation.
-
-This package replaces the [System.Numerics.Tensors package](https://www.nuget.org/packages/System.Numerics.Tensors).
+AI workloads like semantic search and retrieval-augmented generation (RAG) extend the natural language capabilities of large language models, such as ChatGPT, by augmenting prompts with relevant data. For these workloads, operations on vectors&mdash;like *cosine similarity* to find the most relevant data to answer a question&mdash;are crucial. The System.Numerics.Tensors.TensorPrimitives package provides APIs for vector operations.
```
Comment (richlander) [inline docs/core/whats-new/dotnet-8/runtime.md:780]: > System.Numerics.Tensors.TensorPrimitives

Is that new? I don't see it on nuget.org currently.

**Review by richlander — APPROVED** [review]
Thanks!

**Review by IEvangelist — APPROVED** [review]

File: docs/core/whats-new/dotnet-8/runtime.md (hunk 1 above)
Comment (stephentoub) [reply docs/core/whats-new/dotnet-8/runtime.md:780]: The library / package is System.Numerics.Tensors:
https://www.nuget.org/packages/System.Numerics.Tensors

The type in that package is System.Numerics.Tensors.TensorPrimitives.

So the phrase "The System.Numerics.Tensors.TensorPrimitives package" should be "The System.Numerics.Tensors.TensorPrimitives type".

Comment (stephentoub) [inline docs/core/whats-new/dotnet-8/runtime.md:780]:
```suggestion
AI workloads like semantic search and retrieval-augmented generation (RAG) extend the natural language capabilities of large language models, such as ChatGPT, by augmenting prompts with relevant data. For these workloads, operations on vectors&mdash;like *cosine similarity* to find the most relevant data to answer a question&mdash;are crucial. The System.Numerics.Tensors.TensorPrimitives type provides APIs for vector operations.
```

**Pull request merged**